index.bs

```diff
@@ -839,6 +841,27 @@ When this method is invoked, the user agent MUST execute the following algorithm
                 1.  Let |attestationObject| be a new {{ArrayBuffer}}, created using |global|'s [=%ArrayBuffer%=], containing the
                     bytes of <code>|credentialCreationData|.[=attestationObjectResult=]</code>'s value.
 
+                1.  If <code>|credentialCreationData|.[=AttestationConveyancePreferenceOption=]</code>'s value is
+                    <dl class="switch">
+                        :   "none"
+                        ::  replace potentially uniquely identifying information (such as [=AAGUID=] and 
+                            [=attestation certificates=]) in the [=attested credential data=] and [=attestation statement=],
+                            respectively, with blinded versions of the same data.
+                            
+                        Issue: need to specify how to perform such blinding?
```

Comment:
I think we need to define "blinding" rather than necessarily proscribe what to do, but let's do it at CR.

If we get agreement on this PR, let's follow up and add an issue number into this line.

Reply:
done  in 802883f